.env.startup

```
@@ -0,0 +1,47 @@
+# Logging Configuration
+LOG_LEVEL=info
+STARTUP_LOG_FILE=/tmp/celebrum-startup.log
+
+# Webhook Configuration
+TELEGRAM_WEBHOOK_URL=
+TELEGRAM_WEBHOOK_SECRET=
+WEBHOOK_TIMEOUT=30
+WEBHOOK_MAX_CONNECTIONS=100
+
+# Service URLs (Internal)
+POSTGRES_URL=postgres://${POSTGRES_USER:?set POSTGRES_USER}:${POSTGRES_PASSWORD:?set POSTGRES_PASSWORD}@postgres:5432/${POSTGRES_DB:?set POSTGRES_DB}
```

dotenv-linter (3.3.0)

[warning] [SubstitutionKey] The POSTGRES_URL key is not assigned properly

(SubstitutionKey)